### PR TITLE
13 section book 56

### LINKED ISSUE #176: Section 13 Book 1
```
‘The Violet Ocean’s a dangerous place, Cap’n’, says the first
mate. ‘The crew probably won’t follow you there if they don’t
think you’re good enough!’
If your Rank is 4 or more, turn to paragraph 55 in Over the
Blood-Dark Sea. If your Rank is less than 4, the first mate advises
you against the ocean journey. If you take his advice, turn back
to 507.
If you insist on making the trip, you need to make a
CHARISMA roll at Difficulty 12 to convince the crew to follow
you. If you succeed, turn to paragraph 55 in Over the Blood-Dark
Sea. Otherwise, turn back to 507.
```

**Reference features required (if any)**
Ranking System
Charisma Roll

**Additional context**

### LINKED ISSUE #232: Section 33 Book 1
```
Resurrection costs 200 Shards if you are an initiate, and 800
Shards if not. Once you have arranged for resurrection you need
not fear death, as you will be magically restored to life here at
the temple. To arrange resurrection, pay the fee and write
‘Temple of Tyrnai, The War-Torn Kingdom 640’ in the
Resurrection box on your Adventure Sheet. If you are later
killed, turn to 640 in The War-Torn Kingdom.
You can have only one resurrection arranged at any one
time. If you arrange another resurrection later at a different
temple, the original one is cancelled. Cross it off your Adventure
Sheet. You do not get a refund.
When you are finished here, turn to 282.
```

**Reference features required (if any)**
#10 
**Ressurection** needed

**Additional context**

### PR DESCRIPTION
**Pull request by Lion-alt**
### What?
Added implementable sections
### How?
Added more Java Sections
### Anything Else?
closes #232 
closes #176

### CODE REVIEW

File: fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/book/book1/Section33.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                Once you have arranged for resurrection you need
+                not fear death, as you will be magically restored to life here at
+                the temple. To arrange resurrection, pay the fee and write
+                ‘Temple of Tyrnai, The War-Torn Kingdom 640’ in the
+                Resurrection box on your Adventure Sheet. If you are later
+                killed, turn to 640 in The War-Torn Kingdom.
+                You can have only one resurrection arranged at any one
+                time. If you arrange another resurrection later at a different
+                temple, the original one is cancelled. Cross it off your Adventure
+                Sheet. You do not get a refund.""").activeIf(current.shards() >= 200 && isInitiate,x -> x.clickable( n -> n.subtractStamina(200).withResurrectionArrangement(new Resurrection("Temple of Tyrnai, The War-Torn Kingdom 640",SectionId.book1(640))), c -> c.text("Resurrection costs 200 Shards if you are an initiate.").clickableTurnTo(282)))
```
Comment (Nilstrieb) [inline fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/book/book1/Section33.java:42]: can you fix the formatting here, this is not very readable
automated Java formatting in CI would still be appreciated but I haven't looked into it much

**Review by Desi7 — APPROVED** [review]
looking good 👍

**Review by adrsch04 — APPROVED** [review]
Looks legitimate

**Review by Nilstrieb — APPROVED** [review]
lets stop with the java sections and do yaml sections instead, doesnt matter if theyre wrong

**Review by dbabvwid — APPROVED** [review]

**Review by elion004 — APPROVED** [review]
lgtm

**Pull request merged**